src/pls/models/node.py

```diff
@@ -243,8 +244,9 @@ def table_row(self) -> Union[dict[str, str], None]:
         if args.details:
             cells["type"] = self.type_char
             cells["perms"] = get_permission_text(self.stat.st_mode)
-            cells["user"] = get_user(self.stat.st_uid)
-            cells["group"] = get_group(self.stat.st_gid)
+            if platform != "win32":
+                cells["user"] = get_user(self.stat.st_uid)
+                cells["group"] = get_group(self.stat.st_gid)
```

Comment:
The values returned by the functions will be `None` on Windows, can skip the platform check here.